Fix same name as collection (#353)
tinacms/tina-graphql-gateway: .changeset/poor-pugs-decide.md

---
'tina-graphql': patch
---

> Fixes an issue where mutation payloads got mixed up when the collection name was the same as a name of one of it's fields

Reviewer: ```suggestion
Fixes an issue where mutation payloads got mixed up when the collection name was the same as the name of one of its fields
```
